omahelsinki/urls.py

```diff
@@ -21,6 +21,7 @@
     path('', include('helusers.urls')),
     path('mydata/', TemplateView.as_view(template_name='react_base.html'), name='mydata'),
     path('welcome/', TemplateView.as_view(template_name='react_base.html'), name='welcome'),
+    re_path(r'^app/', TemplateView.as_view(template_name='react_base.html'), None, name='app'),
```

Comment:
What's up with `None` there? Why do you need it?

Reply:
ah, so used to php so I thought you had to provided a default value if you omitted on.
The parameters seems to be named so I aguess you can just omit it?

`re_path(r'^app/', TemplateView.as_view(template_name='react_base.html'), name='app')` ?

Reply:
yes, you can omit "non-named" parameters :)